src/sentry/api/endpoints/project_users.py

```diff
@@ -42,3 +43,30 @@ def get(self, request, project):
             paginator_cls=DateTimePaginator,
             on_results=lambda x: serialize(x, request.user),
         )
+
+    def delete(self, request, project):
+        """
+        Delete an Event User
+        ````````````````````````````````
+
+        Delete an event's user.
+
+        :pparam string organization_slug: the slug of the organization.
+        :pparam string project_slug: the slug of the project.
+        :qparam string query: Prefix ``id`` should be used to match on.
+                              For example, ``query=id:3``
+        """
+        if is_active_superuser(request):
+            identifier = [v for k, v in request.query_params.items()]
+            pieces = identifier[0].strip().split(":", 1)
+            queryset = EventUser.objects.filter(project_id=project.id, ident=pieces[1])
```

Comment:
This allows arbitrary queries to be built. You should have an allow list of fields you expect people to delete with. While this model doesn't have any relationships, in other models you could use this to traverse into associations, or construct invalid queries that cause 500s